src/screens/earn/main-info/main-info.tsx

```diff
@@ -66,7 +67,12 @@ export const MainInfo: FC = () => {
     return result;
   }, [farms]);
 
-  const areAnyRewards = totalClaimableRewardsInUsd.isGreaterThan(DEFAULT_AMOUNT);
+  const areAllRewardsClaimable = useMemo(() => {
```

Comment:
As I understood, a user has to wait now for stakes across all farms to pass the due date, so they can't harvest full rewards in some cases

Reply:
Yep